pkg/render/node_test.go

```diff
@@ -652,8 +663,11 @@ var _ = Describe("Node rendering tests", func() {
 		amazonVPCInstalllation := &operator.Installation{
 			Spec: operator.InstallationSpec{
 				KubernetesProvider: operator.ProviderEKS,
-				CNI:                &operator.CNISpec{Type: operator.PluginAmazonVPC},
-				FlexVolumePath:     "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
+				CNI: &operator.CNISpec{
+					Type:     operator.PluginAmazonVPC,
+					LogLevel: "INFO",
```

Comment:
I'd expect this to be an invalid config (we don't support the LogLevel field with non-Calico CNI plugins).

That said, I think we should remove LogLevel from this PR for now while we figure out how we want to expose CNI config more generally